gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -83,6 +84,7 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
         const val KSP_GROUP_ID = "com.google.devtools.ksp"
         const val KSP_PLUGIN_CLASSPATH_CONFIGURATION_NAME = "kspPluginClasspath"
         const val KSP_PLUGIN_CLASSPATH_CONFIGURATION_NAME_NON_EMBEDDABLE = "kspPluginClasspathNonEmbeddable"
+        val LANGUAGE_VERSION = KotlinVersion.KOTLIN_1_8
```

Comment:
You can use `org.jetbrains.kotlin.config.LanguageVersion.LATEST_STABLE` until bumping to 2.0.

Reply:
LATEST_STABLE is 1.9 in main branch.